package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Fri Feb 23 14:57:51 UTC 2018 - jreidinger@suse.com
+
+- Do not allow to encrypt too small partition (bsc#1065071)
+- Check that if separated "/boot" is used that it is big enough
```

Comment:
NP: I see it a bit difficult to read. Probably some like this?: `- Check size for separate /boot`.